cli.js

```diff
@@ -1,60 +1,64 @@
 #!/usr/bin/env node
 
-var pipboylib = require('pipboylib')
-var relay = require('pipboylib/lib/relay')
+var pipboylib = require('pipboylib');
+var relay = require('pipboylib/lib/relay');
```

Comment:
Bring on the semi-colons!

I'm a bigger fan of them existing since that's how people tend to learn JavaScript and folks don't need to understand automatic semicolon insertion. Sorry about starting us off on a polar topic. :smile: